src/components/CompleteProfile/CompleteProfile.tsx

```diff
@@ -0,0 +1,419 @@
+										<label htmlFor='day'>Day</label>
+										<select id='day' name='day'>
+											<option selected disabled hidden value=''>Select a day</option>
+											<option value='01'>1</option>
+											<option value='02'>2</option>
+											<option value='03'>3</option>
+											<option value='04'>4</option>
+											<option value='05'>5</option>
+											<option value='06'>6</option>
+											<option value='07'>7</option>
+											<option value='08'>8</option>
+											<option value='09'>9</option>
+											<option value='10'>10</option>
+											<option value='11'>11</option>
+											<option value='12'>12</option>
+											<option value='13'>13</option>
+											<option value='14'>14</option>
+											<option value='15'>15</option>
+											<option value='16'>16</option>
+											<option value='17'>17</option>
+											<option value='18'>18</option>
+											<option value='19'>19</option>
+											<option value='20'>20</option>
+											<option value='21'>21</option>
+											<option value='22'>22</option>
+											<option value='23'>23</option>
+											<option value='24'>24</option>
+											<option value='25'>25</option>
+											<option value='26'>26</option>
+											<option value='27'>27</option>
+											<option value='28'>28</option>
+											<option value='29'>29</option>
+											<option value='30'>30</option>
+											<option value='31'>31</option>
+										</select>
+									</div>
+								</div>
+								<div className='slider-wrapper'>
+									<div className='slider-checkbox-row'>
+										<input
+											id='isBasedOnGTA'
+											name='isBasedOnGTA'
+											type='checkbox'
+											className='slider-checkbox'
+										/>
+										<label htmlFor='isBasedOnGTA'>
+											I'm based in Toronto or Greater Toronto Area
+										</label>
+									</div>
+									<div className='slider-checkbox-row'>
+										<input
+											id='canJoinLocalEvents'
+											name='canJoinLocalEvents'
+											type='checkbox'
+											className='slider-checkbox'
+										/>
+										<label htmlFor='canJoinLocalEvents'>
+											I can join TorontoJS's local events
+										</label>
+									</div>
+								</div>
+							</div>
+						</div>
+					</details>
+
+					{/* TODO: Need to confirm adding green check icon on successful section completion on mobile view */}
+					<details open>
+						<summary className='text-h6'>
+							<span>Avatar:</span>
+						</summary>
+
+						<div className='details-content-wrapper'>
+							<div className='details-content-file-upload'>
+								{photoFile && (
+									<picture>
+										<img src={photoFile} />
+									</picture>
+								)}
+
+								<div className='details-file-upload-buttons-wrapper'>
+									{photoFile && (
+										<span className='file-upload-success'>
+											Avatar uploaded successfully
+										</span>
+									)}
```

Comment:
```suggestion
									<span ref={avatarUploadStatusRef} aria-live='polite' aria-atomic='true' role='status' className='file-upload-success'>
										{photoFile ? 'Avatar uploaded successfully' : ''}
									</span>
```

This will make the upload be announced (or should make it so). And of course, we need to add a ref here because react is a piece of dumpster fire